arkouda/util.py

```diff
@@ -225,6 +225,8 @@ def attach(name: str, dtype: str = "infer"):
     Attaches to a known element name. If a type is passed, the server will use that type
     to pull the corresponding parts, otherwise the server will try to infer the type
     """
+    from arkouda.infoclass import list_registry
+
```

Comment:
Is this here for circular dependency reasons? if not we should move it to the top with the rest of the imports